src/nyc_trees/apps/users/urls/group.py

```diff
@@ -34,11 +34,10 @@
         name='edit_user_mapping_priveleges'),
 
     # Keep in sync with src/nyc_trees/js/src/reserveBlockfacePage.js
-    url(r'^(?P<group_slug>[\w-]+)/request-individual-mapper-status/$',
+    url(r'^(?P<group_slug>[\w-]+)/request-trusted-mapper-status/$',
```

Comment:
The ORM model for group "individual mappers" is called TrustedMapper so this new name should be more accurate. This particular change isn't related to the global "individual mappers" concept but I lumped it together with the other changes anyway.